src/displayapp/screens/SystemInfo.cpp

```diff
@@ -12,15 +12,16 @@
 #include "drivers/Watchdog.h"
 
 using namespace Pinetime::Applications::Screens;
+using DeviceTypes = Pinetime::Controllers::MotionController::DeviceTypes;
```

Comment:
If you're doing `using Foo = Some::Namespaces::Foo;` you can just do `using Some::Namespaces::Foo;` instead.

Reply:
I can't because `Pinetime::Controllers::MotionController` is not a namespace or unscoped enum.